Explain usage of deployment API tokens (#3167)
astronomer/docs: astro/deployment-api-tokens.md

1. In the Cloud UI, open your Workspace, then open the Deployment that the API token belongs to.

2. Click **Edit** next to your API token.

3. Click **Rotate token**. The Cloud UI rotates the token and shows the new token value.

4. Copy the new token value and store it in a safe place. You will not be able to retrieve this value from Astro again.

5. In any workflows using the token, replace the old token value with the new value you copied.

## Use a Deployment API token with the Astro CLI

To use a Deployment API token with Astro CLI, specify the `ASTRO_API_TOKEN` environment variable in the system running the Astro CLI:

```sh
export ASTRO_API_TOKEN=<your-token>
```

After you configure the `ASTRO_API_TOKEN` environment variable, you can run Astro CLI commands related to the Deployment for which the Deployment API token was created. For example, [`astro deployment inspect`](cli/astro-deployment-inspect.md) or [`astro deployment logs`](cli/astro-deployment-logs.md).

:::info

> If you have configured both a Deployment API token using `ASTRO_API_TOKEN` and Deployment API keys (deprecated) with `ASTRONOMER_KEY_ID`/`ASTRONOMER_KEY_SECRET`, the Deployment API token takes precedence.

Reviewer: ```suggestion
If you have configured both a Deployment API token using `ASTRO_API_TOKEN` and Deployment API keys (deprecated) with `ASTRONOMER_KEY_ID` or `ASTRONOMER_KEY_SECRET`, the Deployment API token takes precedence.
```

4. Copy the new token value and store it in a safe place. You will not be able to retrieve this value from Astro again. 

5. In any workflows using the token, replace the old token value with the new value you copied.

## Use a Deployment API token with the Astro CLI

To use a Deployment API token with Astro CLI, specify the `ASTRO_API_TOKEN` environment variable in the system running the Astro CLI:

```sh
export ASTRO_API_TOKEN=<your-token>
```

> After you configure the `ASTRO_API_TOKEN` environment variable, you can run Astro CLI commands related to the deployment for which the Deployment API token was created. For example, `astro deployment inspect` or `astro deployment logs`.

Reviewer: ```suggestion
After you configure the `ASTRO_API_TOKEN` environment variable, you can run Astro CLI commands related to the Deployment for which the Deployment API token was created. For example, `astro deployment inspect` or `astro deployment logs`.
```

:::info

> If you have configured both a Deployment API token via `ASTRO_API_TOKEN` and Deployment API keys (deprecated) via `ASTRONOMER_KEY_ID`/`ASTRONOMER_KEY_SECRET`, the Deployment API token takes precedence. 

Reviewer: ```suggestion
If you have configured both a Deployment API token using `ASTRO_API_TOKEN` and Deployment API keys (deprecated) with `ASTRONOMER_KEY_ID`/`ASTRONOMER_KEY_SECRET`, the Deployment API token takes precedence. 
```
